beancount_import/source/schwab_csv.py

```diff
@@ -1077,10 +1319,10 @@ def _load_positions_csv(
         symbol = STRIP_FROM_SYMBOL_RE.sub("", symbol)
         quantity = _convert_int(row["Quantity"])
```

Comment:
The quantity of reinvested portfolios can be decimal. I need to replace this line with `quantity = _convert_decimal(row["Quantity"])`.